spec/support/mockers/category.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
Were some of these files already merged in other PR's (or are about to be)? In other words, are there changes here from the mockers that have already been reviewed?